jobs/worker.go

```diff
@@ -91,3 +98,48 @@ func RunJobNow(j Job) {
 	}
 	l.Log.Infof("Finished Job %v with %v", j.Name(), j.Arguments())
 }
+
+// BackgroundWorkerHealthChecker is a struct that contains the time for the health checker and a channel to pass the shared memory for the time through time channel
+type BackgroundWorkerHealthChecker struct {
+	timeStamp   time.Time
+	timeChannel chan time.Time
+}
+
+func (hc *BackgroundWorkerHealthChecker) healthChecker() {
+	//run healthcheck loop and consumer in the background
+	go hc.healthcheckLoop()
+	go hc.healthCheckConsumer()
+
+	//create new GET request through echo with the same path name "/health"
+	e := echo.New()
+	e.HideBanner = true
+	e.HidePort = true
+	e.GET("/health", func(c echo.Context) error {
+		//health should return a 500 if there hasn't been a response from ping request within 30 seconds
+		if hc.timeStamp.Before(time.Now().Add(-30 * time.Second)) {
+			return c.String(http.StatusInternalServerError, "500")
```

Comment:
And one last comment here - can we change the `"500"` message to `"Failed to hit redis for more than 30 seconds"`?